test/functional/ui/screen_basic_spec.lua

```diff
@@ -73,33 +73,29 @@ describe('Screen', function()
   describe(':suspend', function()
     it('is forwarded to the UI', function()
       local function check()
-        if not screen.suspended then
-          return 'Screen was not suspended'
-        end
+        eq(true, screen.suspended)
       end
```

Comment:
> So the only issue is that screen:expect() loses the distinction between errors and failures? 

No, I don't care about that. 

Nothing blocking this. I also just tried `helpers.retry()` with an `eq()` assertion and it worked. So I must have been confused.